src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Red Hat OpenShift
+description: How to install Okteto onto Red Hat OpenShift
+sidebar_label: Red Hat OpenShift
+id: openshift
+pagination_prev: get-started/install/index
+pagination_next: get-started/install-okteto-cli
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+
+This guide will walk you through the process of installing Okteto in Red Hat OpenShift.
+
+## Installation Requirements
+
+Before you start, make sure you have the following CLIs installed in your machine:
+
+- `kubectl` >= 1.28 ([kubectl installation guides](https://kubernetes.io/docs/tasks/tools/#kubectl))
+- `helm` >= 3.14 ([helm installation guides](https://helm.sh/docs/intro/install/))
+- `oc` >= 4.0.0 
+
+You'll also need the following:
+
+- An Okteto License
+- A Kubernetes cluster with OpenShift (≥ 4.0.0) installed
+
+For this guide, you will need to obtain the following values:
+
+- `<your-openshift-subdomain>`: for example, `okteto.r9yz.p2.openshiftapps.com`
+- `<cluster-name>`: for example, `okteto`
+
+## Getting your Okteto License
+
+A license is mandatory to use Okteto. You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
+
+If you are interested in evaluating Okteto, [sign up for our free 30 days trial](https://www.okteto.com/free-trial/). No credit card required.
+
+## Steps before Installing Okteto
+
+Before installing Okteto, you will need to follow these steps:
+
+- Create the `okteto` namespace
+- Allow containers as `root` in the `okteto` namespace
+- Allow Okteto Build service privileged access
+- Allow Okteto Daemon service to access the host
+- Copy Openshift wildcard certificate to the `okteto` namespace
+- Get the OpenShift Ingress Controller IP
+
+### Create the `okteto` namespace
+
+Run the following command to create the `okteto` namespace:
+
+```bash
+kubectl create namespace okteto
+```
+
+### Allow containers as `root` in the `okteto` namespace
+
+By default, running `root` containers in OpenShift is not allowed. Today, most of the Okteto Control Plane runs as `root` (this is something we will fix soon). To allow `root` containers for the Okteto Control Plane components, run the following commands:
+
+```bash
+oc adm policy add-scc-to-user anyuid -z default -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-buildkit -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-registry -n okteto 
+```
+
+### Allow Okteto Build service privileged access
```

Comment:
This "reads" odd. Is this the terminology that openshift uses for this action?

Reply:
`Looking at this doc`, it looks like the wording they use is "Grant access to the  "privileged" SCC. 

Also, we are asking them to grant access to the Service Account right? I think using `service` might confuse them since `service` means something else in Kubernetes.